erpnext/stock/dashboard/item_dashboard.js

```diff
@@ -16,18 +16,6 @@ erpnext.stock.ItemDashboard = Class.extend({
 		this.content = $(frappe.render_template('item_dashboard')).appendTo(this.parent);
 		this.result = this.content.find('.result');
 
-		// move
-		this.content.on('click', '.btn-move', function() {
-			erpnext.stock.move_item(unescape($(this).attr('data-item')), $(this).attr('data-warehouse'),
```

Comment:
create new `Stock Entry` with Item and Warehouse set (?)

Reply:
Made the changes suggested.